app/potpie.py

```
@@ -0,0 +1,628 @@
+    """Talk with your conversations"""
+    pass
+
+
+@conversation.command()
+@click.argument("title")
+def create(title):
+    """Create a new conversation"""
+    base_url = "http://localhost:8001"
+
+    # Sees that user_id is used as the defaultUsername
+    user_id = os.getenv("defaultUsername", "defaultuser")
```

Ruff (0.8.2)

Use capitalized environment variable `DEFAULTUSERNAME` instead of `defaultUsername`

(SIM112)